layouts/partials/comments/provider/cactus.html

```diff
@@ -0,0 +1,13 @@
+{{- with .Site.Params.comments.cactus -}}
+<div id="comment-section"></div>
+
```

Comment:
It seems that JS is not loaded.
Please write style in html.

Reply:
Oh it Looks like while copying it over from my Blog I missed to add the required js file to the head here. Will fix that :)